packages/core/admin/admin/src/content-manager/history/components/tests/VersionHeader.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import { UID } from '@strapi/types';
+import { render as renderRTL, screen } from '@tests/utils';
+import { Route, Routes } from 'react-router-dom';
+
+import { VersionHeader } from '../VersionHeader';
+
+// Mocks
+const layout = {
+  contentType: {
+    info: {
+      singularName: 'kitchensink',
+    },
+    settings: {
+      mainField: 'title',
+    },
+  },
+};
+const version = {
+  id: '26',
+  contentType: 'api::kitchensink.kitchensink' as UID.ContentType,
+  relatedDocumentId: 'pcwmq3rlmp5w0be3cuplhnpr',
+  createdAt: '2022-01-01T00:00:00Z',
+  status: 'draft' as const,
+  schema: {},
+  locale: null,
+  data: {
+    title: 'Test Title',
+  },
+};
+
+const render = (props: React.ComponentProps<typeof VersionHeader>) =>
+  renderRTL(
+    <Routes>
+      <Route
+        path="/content-manager/:collectionType/:slug/:id/history"
+        element={<VersionHeader {...props} />}
+      />
+    </Routes>,
```

Comment:
Are you sure you need to wrap it in react router stuff? I was hoping we could just test the component directly